src/components/UserList/UserList.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import {
+ StyleSheet,
+ Text,
+ View,
+ Navigator,
+ Image,
+ ListView,
+} from 'react-native';
+import Header from './Header';
+import Footer from './Footer';
+import Row from './Row';
+
+const API_KEY = '73b19491b83909c7e07016f4bb4644f9:2:60667290';
+const QUERY_TYPE = 'hardcover-fiction';
+const API_STEM = 'http://api.nytimes.com/svc/books/v3/lists';
```

Comment:
These constants were for networking test purpose. It'll be replaced our back-end API endpoint. will add comment!